<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>frmBottomMenu</class>
 <widget class="QWidget" name="frmBottomMenu">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1024</width>
    <height>89</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QPushButton" name="btnHome">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>10</y>
     <width>46</width>
     <height>73</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="btnWarning">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>10</y>
     <width>46</width>
     <height>73</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="btnProcess">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>10</y>
     <width>46</width>
     <height>73</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="btnStatus">
   <property name="geometry">
    <rect>
     <x>490</x>
     <y>10</y>
     <width>46</width>
     <height>73</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="btnPause">
   <property name="geometry">
    <rect>
     <x>640</x>
     <y>10</y>
     <width>46</width>
     <height>73</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="btnVideo">
   <property name="geometry">
    <rect>
     <x>790</x>
     <y>10</y>
     <width>46</width>
     <height>73</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="btnBack">
   <property name="geometry">
    <rect>
     <x>940</x>
     <y>10</y>
     <width>46</width>
     <height>73</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QWidget" name="widget" native="true">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>0</y>
     <width>21</width>
     <height>89</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">image: url(:/images/split.png);</string>
   </property>
  </widget>
  <widget class="QWidget" name="widget_2" native="true">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>0</y>
     <width>21</width>
     <height>89</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">image: url(:/images/split.png);</string>
   </property>
  </widget>
  <widget class="QWidget" name="widget_3" native="true">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>0</y>
     <width>21</width>
     <height>89</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">image: url(:/images/split.png);</string>
   </property>
  </widget>
  <widget class="QWidget" name="widget_4" native="true">
   <property name="geometry">
    <rect>
     <x>580</x>
     <y>0</y>
     <width>21</width>
     <height>89</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">image: url(:/images/split.png);</string>
   </property>
  </widget>
  <widget class="QWidget" name="widget_5" native="true">
   <property name="geometry">
    <rect>
     <x>730</x>
     <y>0</y>
     <width>21</width>
     <height>89</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">image: url(:/images/split.png);</string>
   </property>
  </widget>
  <widget class="QWidget" name="widget_6" native="true">
   <property name="geometry">
    <rect>
     <x>880</x>
     <y>0</y>
     <width>21</width>
     <height>89</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">image: url(:/images/split.png);</string>
   </property>
  </widget>
  <widget class="QWidget" name="widget_title" native="true">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>10</y>
     <width>133</width>
     <height>15</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-image: url(:/images/bottom_menu_title.png);</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
